pkg/genericapiserver/options/server_run_options.go

```diff
@@ -258,6 +259,11 @@ func (s *ServerRunOptions) AddUniversalFlags(fs *pflag.FlagSet) {
 		"If specified, a username which avoids RBAC authorization checks and role binding "+
 		"privilege escalation checks, to be used with --authorization-mode=RBAC.")
 
+	fs.BoolVar(&s.AnonymousAuth, "anonymous-auth", s.AnonymousAuth, ""+
```

Comment:
Let's make this flag be called "forbid-anonymous-auth", and default to false.

That is because we expect most people to enable it going forward, and we don't want a flag that is expected to be required to be specified.

  The release note can have a warning that: *people who are relying on authentication alone to authorize access should add authorization or set `--forbid-anonymous-auth`*.